src/xdr/Stellar-transaction.x

```diff
@@ -635,7 +635,27 @@ struct TransactionSignaturePayload
 
 /* Operation Results section */
 
-/* This result is used when offers are taken during an operation */
+enum ClaimAtomType
+{
+    CLAIM_ATOM_TYPE_V0 = 0,
+    CLAIM_ATOM_TYPE_ORDER_BOOK = 1
+};
+
+struct ClaimOfferAtomV0
```

Comment:
Reminder to add the comment from the updated CAP-0038 doc - https://github.com/stellar/stellar-protocol/pull/991.

Reply:
Done.